### DISCUSSION TITLE
Anagrafica alunni

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#332] [Idee e proposte]
Buongiorno ,dato che ormai tutti i registri lo riportano ,mi sembra giusto che anche Giuaschool ,che non è da meno lo debba riportare.Ossia sarebbe molto utile se i genitori ,cliccando in alto a destra sul proprio nome e cognome potessero visualizzare tutti i dati anagrafici dei figli e non solo il nome cognome e nome utente , e sarebbe bello se fosse data loro la possibilità di modificare alcuni dati in caso di errori da parte della segreteria.

**Comment by trinko** [comment]
Lo segno come richiesta
